pages/docs/core-concepts/error-handling.mdx

```diff
@@ -152,6 +154,62 @@ const program = pipe(
 )
 ```
 
+#### Mapping over the error channel
```

Comment:
```suggestion
#### Manipulating Expected Errors
```
Was just trying to think of a way to keep this heading relatively consistent with the above heading (i.e. `Catching Expected Errors`).

Reply:
Mine had said "Modifying", but you were right in that you're not technically modifying it, although I had simplified it that way because the error channel changes from that point forward to the new value you mapped it to.

Reply:
Alright I think we can just go with "Manipulating Expected Errors" for now if you're cool with that - can always change it later, but at least this way the headings are consistent.

Reply:
Sure